shell/packages/sandstorm-db/db.js

```diff
@@ -582,6 +582,12 @@ matchWildcardHost = function(host) {
 
   var prefix = wildcardHost[0];
   var suffix = wildcardHost[1];
+
+  // We ignore everything after the first : character to be agnostic
+  // as to what port a request came in on.
+  suffix = suffix.replace(/:.*/, "");
```

Comment:
How about: `.split(":")[0]`

That way I don't have to figure out whether this really needs to be `/:[^:]*$/` or whatever. :)